src/main.ts

```diff
@@ -274,22 +274,15 @@ function getFilepath(
     const files = globSync(globPattern);
 
     const wildcard = globSync(wildcardPattern);
+    const resolvedPath = path.resolve(`${process.cwd()}/${globPattern}`);
```

Comment:
Doesn't `process.cwd()` return a windows-style path on windows? Will that work with a `/` delimiter?

Reply:
Should work cross platform now.